RoboFile.php

```diff
@@ -532,4 +535,12 @@ protected function getPluginVersion($file) {
     preg_match('/^[ \t*]*Version:(.*)$/mi', $data, $m);
     return !empty($m[1]) ? trim($m[1]) : false;
   }
+
+  public function parallelSplitTests() {
+    return $this->taskSplitTestFilesByGroups(4)
+      ->projectRoot('.')
+      ->testsFrom('tests/acceptance')
+      ->groupsTo('tests/acceptance/_groups/group_')
+      ->run();
+  }
```

Comment:
Could this be called `test:acceptance:create-groups` or something like that? The commands are namespaced:
```
Available commands:
  help                        Displays help for a command
  install
  list                        Lists commands
  makepot
  packtranslations
  publish
  pushpot
  qa
  txinit
  update
  watch
 compile
  compile:all
  compile:css
  compile:js
 container
  container:dump
 delete
  delete:docker
 qa
  qa:code-sniffer
  qa:lint
  qa:lint-javascript
  qa:phpstan
 security
  security:composer
 svn
  svn:checkout
  svn:publish
 test
  test:acceptance
  test:acceptance-multisite
  test:coverage
  test:debug-integration
  test:debug-unit
  test:failed-integration
  test:failed-unit
  test:integration
  test:javascript
  test:multisite-integration
  test:unit
 watch
  watch:css
  watch:js
```

Reply:
Done, `test:acceptance-group-tests` - I don't think there can be `:` twice